resources/config.yml

```
@@ -7,9 +7,14 @@ mysql:
   username: root
   password: 
   database: 21C
+  delete: 30 #deletes logs older than X days. Set to 0 to never delete.
```

Use descriptive variable names
```suggestion
  daysToRetainLogs: 30 #deletes logs older than X days. Set to 0 to never delete.
```